lib/palindrome_permutation.rb

```diff
@@ -1,4 +1,21 @@
 
 def palindrome_permutation?(string)
```

Comment:
This works, but it's O(n log n) in time complexity.  Using a hash you can get this to O(n) time complexity which is much better.  

I encourage you to think about how to do this.